scipy/stats/__init__.py

```diff
@@ -429,23 +517,6 @@
    circvar
    circstd
 
-Contingency table functions
----------------------------
-
-.. autosummary::
-   :toctree: generated/
-
-   chi2_contingency
-   contingency.crosstab
-   contingency.expected_freq
-   contingency.margins
-   contingency.relative_risk
-   contingency.association
-   contingency.odds_ratio
```

Comment:
I would put these 3 with the other contingency functions. For `crosstab, expected_freq, margins` these are helper functions and can be set aside. Or in the same section but with a line saying these are utilities.

Reply:
Oh this is what you want to do with the contingency table section you added L443.

Mmm but I think that does not work as in a section named like this I would assume to find the "star" methods.

Reply:
I don't understand.

Reply:
What I mean is to do the following grouping.

```
Here are some contingency table functions...

.. autosummary::
    :toctree: generated/

    chi2_contingency
    fisher_exact
    barnard_exact
    boschloo_exact
    contingency.relative_risk
    contingency.association
    contingency.odds_ratio

And here some helper (could be a subsection)

.. autosummary::
    :toctree: generated/

    contingency.crosstab
    contingency.expected_freq
    contingency.margins

```

Reply:
Ah, yes, splitting `contingency` into sections would be a good follow-up, since it doesn't render very well.

Note that only the ones that are not in`contingency` (chi2_contingency, fisher_exact, barnard_exact, boschloo_exact) are actually hypothesis tests. So here, I've just moved those into the hypothesis test section.